crates/assistant/src/inline_assistant.rs

```diff
@@ -246,17 +246,18 @@ impl InlineAssistant {
             codegen_ranges.push(start..end);
 
             if let Some(model) = LanguageModelRegistry::read_global(cx).active_model() {
-                self.telemetry.report_assistant_event(AssistantEvent {
-                    conversation_id: None,
-                    kind: AssistantKind::Inline,
-                    phase: AssistantPhase::Invoked,
-                    message_id: None,
-                    model: model.telemetry_id(),
-                    model_provider: model.provider_id().to_string(),
-                    response_latency: None,
-                    error_message: None,
-                    language_name: buffer.language().map(|language| language.name().to_proto()),
-                });
+                telemetry::event!(
```

Comment:
@JosephTLyons I think in cases like these where we're reporting the same event in multiple places we probably still should have a struct/function that codifies the shape of the event, otherwise we could end up with disparate data from different locations.